.github/dependabot.yml

```
@@ -0,0 +1,22 @@
+version: 2
+registries:
+  rubygems-server-enterprise-contribsys-com:
+    type: rubygems-server
+    url: https://enterprise.contribsys.com/
+    username: "${{secrets.RUBYGEMS_SERVER_ENTERPRISE_CONTRIBSYS_COM_USERNAME}}"
+    password: "${{secrets.RUBYGEMS_SERVER_ENTERPRISE_CONTRIBSYS_COM_PASSWORD}}"
+updates:
+- package-ecosystem: bundler
+  directory: "/"
+  schedule:
+    interval: weekly
+  ignore:
+    - dependency-name: "vets_json_schema"
+    - dependency-name: "active_model_serializers"
+    - dependency-name: "sidekiq"
+    - dependency-name: "sidekiq-pro"
+    - dependency-name: "sidekiq-ent"
+  open-pull-requests-limit: 5
+  rebase-strategy: disabled
+  registries:
+  - rubygems-server-enterprise-contribsys-com
```

Maybe just name this sidekiq-ent?
```suggestion
  - sidekiq-ent
```